### PR TITLE
DOCS-12782 prereq clarification

### PR DESCRIPTION
**Pull request by wwarner-salesforce**
DOCS-12782 prereq clarification

### CODE REVIEW

**Review by Rachelbot007 — APPROVED** [review]
looks great!

**Review by Rachelbot007 — DISMISSED** [review]
Que bella!

**Review by ferferrarazzo — DISMISSED** [review]

File: modules/ROOT/pages/manual-setup.adoc [hunk 1]
```diff
@@ -136,3 +136,6 @@ Give community users the Exchange viewer permission:
 . Click *Permissions* > *Add permissions* and add the permission *Exchange Viewer*.
 +
 When prompted, select all business groups with any assets that you want community users to be able to view.
+. Manually add the permission *Exchange Viewer* to any users who were invited before you created the team.
+
+After each user logs in to the community, the user is visible in Anypoint.
```
Comment (sathya0) [inline modules/ROOT/pages/manual-setup.adoc]: I remember seeing in the style guide not to use just "Anypoint" but maybe your situation is different?

**Review by sathya0 — DISMISSED** [review]

File: modules/ROOT/pages/manual-setup.adoc (hunk 1 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/manual-setup.adoc]: Good catch! That should be "Anypoint Platform". Fixed. Thank you!

**Review by sathya0 — APPROVED** [review]

**Pull request merged**